test/ese/src/inc/esewrapper.hpp

```diff
@@ -0,0 +1,1478 @@
+#pragma once
```

Comment:
This should have an MIT header right?

Reply:
Ah, I missed *.hpp files

Reply:
By the way has comments, but I actually scanned the file, all comments are benign.  Hopefully the name anonymizer was for sure still run on this file (I didn't see any)?